### ISSUE TITLE
Missing files?

### ISSUE THREAD

**Issue by admarrs** [allenai/vessel-detection-sentinels#6]
I've been trying to test this but have hit a couple of issues. Firstly, the README refers to the script `detect.py` but it seems to be missing. And secondly, when trying to run `main.py` there doesn't appear to be a `config.yaml`.

Can someone point me in the right direction?

**Comment by pbeukema** [comment]
You are correct, we recently refactored this codebase after a major model upgrade and it looks like some portions of the readme are outdated, and as you pointed out, one of the config files was missing. I have created a PR here: https://github.com/allenai/vessel-detection-sentinels/pull/7 with that file and I have started to update the readme. It will take me a little time to go through and remove all of the old comments. In the meantime, if you want to do inference immediately, you can try executing the sample requests in example/s1_request.py and example/s2_request.py for sentinel-1 and sentinel-2 models respectively. Note that will require building and starting the docker container. I appreciate that the documentation for all of this is limited right now. I will update the docs shortly (I am currently out of office).

**Comment by tomADC443** [comment]
PR is merged but the Model Card is still refernecing the detect.py and the updated readme does not explain how to run inference without historical imagery. 


```
6.  Run the inference script for the detection model.

    To run inference on a single image, without access to historical overlaps:

### Sentinel-2
...

```
